src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -50,20 +59,37 @@ private string GetTemplate(string templateName)
             return templateText.ToString();
         }
 
-        private static bool SendMail(string fromEmail, string toEmail, string subject, string body, string cc = "", string bcc = "") //TODO
+        private async Task<bool> SendMailAsync(string fromEmail, string toEmail, string subject, string body, string cc, string bcc)
         {
+            var emailObject = new Email { FromEmail = fromEmail, ToEmail = toEmail, Subject = subject, Body = body };
+
+            var url = "https://jsonplaceholder.typicode.com/posts";
+
+
+            var client = new RestClient(url);
+            var request = new RestRequest(url, Method.Post); 
+            request.AddHeader("Content-Type", "application/json");
+            var message = JsonConvert.SerializeObject(emailObject);
+            request.AddBody(message, "application/json");
+            RestResponse response = await client.ExecuteAsync(request);
+
+            if (response is null)
+                return false;
+            
             return true;
+
         }
 
-        public bool SendActivationEmail(string fromEmail, string toEmail, string templateName, string RecipientName, string subject = "Account Activation", string cc = "", string bcc = "")
+        public async Task<bool> SendActivationEmail(string userId, string fromEmail, string toEmail, string templateName, string RecipientName,
+            string subject = "Account Activation", string cc = "", string bcc = "")
         {
-            var baseUrl = GetBaseUrl();
-            var activationUrl = GetActivationUrl();
+
+            var activationUrl = await GetActivationUrl(userId);
```

Comment:
I noticed you took out the line that strips any potential white spaces off the beginning and end of the template name. Is there a reason why?